src/pages/index.js

```diff
@@ -78,19 +104,129 @@ editProfileButton.addEventListener("click", () => {
   profilePopup.open();
 });
 
-//add new card
+profileImageElement.addEventListener("click", () => {
+  imageChangePopup.open();
```

Comment:
You need to call `resetValidation` for the avatar on opening as well to disable the button